src/code/graph.c

```diff
@@ -166,36 +174,41 @@ void Graph_TaskSet00(GraphicsContext* gfxCtx) {
         osSyncPrintf(VT_FGCOL(RED));
         osSyncPrintf("RCPが帰ってきませんでした。"); // "RCP did not return."
         osSyncPrintf(VT_RST);
+
         LogUtils_LogHexDump((void*)&HW_REG(SP_MEM_ADDR_REG, u32), 0x20);
         LogUtils_LogHexDump((void*)&DPC_START_REG, 0x20);
         LogUtils_LogHexDump(gGfxSPTaskYieldBuffer, sizeof(gGfxSPTaskYieldBuffer));
 
         SREG(6) = -1;
-        if (D_8012D260 != NULL) {
+        if (sPrevTaskWorkBuffer != NULL) {
             HREG(80) = 7;
             HREG(81) = 1;
             HREG(83) = 2;
-            D_8012D260 = D_8012D260;
-            Graph_DisassembleUCode(D_8012D260);
+            sPrevTaskWorkBuffer = sPrevTaskWorkBuffer;
+            Graph_DisassembleUCode(sPrevTaskWorkBuffer);
         }
         Fault_AddHungupAndCrashImpl("RCP is HUNG UP!!", "Oh! MY GOD!!");
     }
 
     osRecvMesg(&gfxCtx->queue, &msg, OS_MESG_NOBLOCK);
 
-    D_8012D260 = gfxCtx->workBuffer;
+    sPrevTaskWorkBuffer = gfxCtx->workBuffer;
     if (gfxCtx->callback != NULL) {
         gfxCtx->callback(gfxCtx, gfxCtx->callbackParam);
     }
 
-    time = osGetTime();
-    if (D_8016A550 != 0) {
-        D_8016A558 = (D_8016A558 + time) - D_8016A550;
-        D_8016A550 = time;
+    timeNow = osGetTime();
+    if (gAudioThreadUpdateTimeStart != 0) {
+        // The audio thread audio update is running on the audio thread
```

Comment:
I feel like this would be better as simply `The audio thread update is running` or `Audio update is running on the audio thread`

Reply:
I think I wanted to try make the "concurrency" (it's not really, only one thread is ever running at the same time) aspect clearer but yeah it can be simplified to `The audio thread update is running`